tests/rustdoc-ui/unused-extern-crate.rs

```diff
@@ -1,3 +1,3 @@
 //@ check-pass
 //@ aux-crate:panic_item=panic-item.rs
-// @has unused_extern_crate/index.html
+//####@ has unused_extern_crate/index.html
```

Comment:
Not sure what to do about this one :\
This test is run in `ui` mode, and as such, doesn't recognize any `htmldocck` directives...

Reply:
There are a couple of options, the way I see it:
1. Just remove the `has` directive. It hasn't actually been effective in a while, and according to some my limited archeological excavation, seems this test was created in response to an ICE that was solved, so `check-pass` should be enough for making sure this doesn't regress.
2. Make this a `rustdoc`-mode test? Not sure if it supports auxiliary crates.
3. Run `htmldocck` for `ui`-mode tests? Doesn't sound feasible, nor does it make much sense.

Tagging @GuillaumeGomez and @jyn514 because you two seem to have worked on this test/issue (cc #68427 , I think?).

Reply:
this should probably have been a `rustdoc` test to start, not rustdoc-ui

Reply:
i would be ok with either moving it to `rustdoc` or just deleting the `@has` directive. like you said, it's just testing that we don't ICE.

Reply:
Cool, thanks for taking a look!
Moved the test to `rustdoc`, arbitrarily picked one of the two options 😅